core/dbt/adapters/base/impl.py

```diff
@@ -53,21 +53,6 @@ def _expect_row_value(key: str, row: agate.Row):
     return row[key]
 
 
-def _relations_filter_schemas(
```

Comment:
I'm confused! Were we not using `_relations_filter_schemas` before? I don't see where the call to this method was removed in this diff??

Reply:
That's kind of a trick question - I removed it because there is no call to this function!

```
$ rg -F _relations_filter_schemas
$ echo $?
1
```

Reply:
huh, ok that was what i was seeing, but I was _pretty sure_ that the `decimal.Decimal does not have any attribute lower` was coming from this method! Was it instead coming from somewhere else? Or did that problem just dissolve with this updated implementation?

Reply:
I think in 0.16.0+ it would be in `_catalog_filter_schemas`

Reply:
ship it, thanks